Checkbox/Checkbox.js

```diff
@@ -162,7 +163,7 @@ const CheckboxBase = kind({
 				disabled={disabled}
 				role="checkbox"
 			>
-				<div className={css.bg} />
+				<div className={classnames(css.bg, (typeof ENACT_PACK_NO_ANIMATION !== 'undefined' && ENACT_PACK_NO_ANIMATION) ? css.noAnimation : null)} />
```

Comment:
I think it would be better if we define bg classnames in `compute`.
So in computed,
```
computed: {
		className: ({indeterminate, selected, standalone, styler}) => styler.append({
			selected,
			standalone,
			indeterminate,
			noAnimation: (typeof ENACT_PACK_NO_ANIMATION !== 'undefined' && ENACT_PACK_NO_ANIMATION)
		}),
```
and in the less file,

```
// Focus States
@checkbox-focus: {
	.checkbox-states(
		// Gathered rules into a rules-set to include the .bg selector in the resting block
		{
			...
			.bg {
				.sand-spotlight-focus-bg-colors();
			}
			&.noAnimation .bg {
				.sand-spotlight-focus-bg-colors(@shadow: {
					box-shadow: none;
				});
			}
		};
```

Reply:
ok